data/model/namespacequota.py

```diff
@@ -137,6 +137,16 @@ def get_namespace_quota_limit(quota, limit_id):
         return None
 
 
+def get_namespaces_with_quotas():
+    namespaces_with_quotas = (
+        UserOrganizationQuota.select(User.id, User.username, UserOrganizationQuota.limit_bytes)
+        .join(User)
+        .dicts()
```

Comment:
These and the other queries are going to be doing full table scans. This is going to put a lot of pressure on the database. Can we [paginate these calls to improve performance?](https://github.com/quay/quay/blob/master/data/model/modelutil.py#L7)